tests/tests_ntp_provider6.yml

```diff
@@ -0,0 +1,72 @@
+
+- name: Configure NTP with OS release non-default provider and then change it to the default provider
+  hosts: all
+  vars:
+    is_dist: "{{ ansible_distribution in ['RedHat', 'CentOS'] }}"
+    is_vers: "{{ ansible_distribution_version is version_compare('7.0', '<') }}"
+    both_avail: true
+
+  tasks:
+  - name: Check for availability of both NTP providers
+    package: name={{ item }} state=present
+    register: package_install
+    ignore_errors: yes
+    with_items:
+      - chrony
+      - ntp
+
+  - set_fact:
```

Comment:
A reference to the Ansible bug in a comment would be nice to explain why `with_items` cannot be used directlly,